Improved hello world page

Himanshu-Mishr opened the pull request:
Hi,
I have added some missing configs and corrected hello world page. More details below. 
1. css loader config in webpack configuration, which will allow to import css in js/jsx files
2. .gitignore
3. updated package.json with packages required to run project.

Let me know if something is not in order. 

Cheers

beven commented:
@Himanshu-Mishr sorry but I had to remove some of your changes in App.jsx because other PRs came before yours - let me know if you want to add your changes back in with another PR. Thanks

Himanshu-Mishr commented:
@beven Thanks for merge. But I have some points to make.

1.   I see that your `Header` and `Content` are functional component. So I changed them from class to function as these components are not using additional features such as local state and lifecycle hooks. These changes were removed just because someone else made PR before!!

2.  I added `./App.css` stylesheet for App component. But I see that my classes are overwritten to some CSS rules which are actually written in `./index.html` file. How is this better and acceptable ?
